pysam/libctabix.pyx

```diff
@@ -72,7 +72,7 @@ from pysam.libchtslib cimport htsFile, hts_open, hts_close, HTS_IDX_START,\
     BGZF, bgzf_open, bgzf_dopen, bgzf_close, bgzf_write, \
     tbx_index_build, tbx_index_load, tbx_itr_queryi, tbx_itr_querys, \
     tbx_conf_t, tbx_seqnames, tbx_itr_next, tbx_itr_destroy, \
-    tbx_destroy, hisremote
+    tbx_destroy, hisremote, region_list
```

Comment:
What's the deal with `region_list`?  It's newly imported here but commented out later.